src/setupTests.ts

```diff
@@ -0,0 +1,4 @@
+import * as matchers from '@testing-library/jest-dom/matchers';
+import { expect } from 'vitest';
+
+expect.extend(matchers);
```

Comment:
Use just import '@testing-library/jest-dom'; instead of configuration above to avoid issues with some assertions (like .toBeInTheDocument())

Reply:
Done